install.sh

```diff
@@ -37,7 +37,7 @@ apt-get install -y \
   openssh-client=1:7.2p2-4ubuntu2.1 \
   openssl=1.0.2g-1ubuntu4.6 \
   software-properties-common=0.96.20.7 \
-  sudo=1.8.16-0ubuntu1.3  \
+  sudo=1.8.16-0ubuntu1.4  \
```

Comment:
Upgraded the sudo version due to [this bug](https://bugs.launchpad.net/ubuntu/+source/sudo/+bug/1688034).